### DISCUSSION TITLE
Why couldn't i reduce the FOT?

### DISCUSSION THREAD

**Discussion by addda3786** [vercel/community#7420] [Help]
### Summary

the report is last 7 days info with a next.js project
FDT 14.09 GB, FOT 57.64 GB
which i have done:
1. access assets file from other server(image / video)
2. less use with serverless function, there is no api service in this project, only the path /news/categoryID/itemID with a server render call api first.

i'm not sure if the middleware with next-i18n-router could make such a huge amount FOT cost.

i also have another project with nuxt.js
FDT 13.08 GB, FOT 2.11MB

Perhaps the official can explain why there is such a significant difference in charges for projects with minimal differences under different frameworks.

![config](https://github.com/vercel/community/assets/174792873/59b875c8-4bd2-4a1a-acbe-fa0bf184ff5f)
![fot](https://github.com/vercel/community/assets/174792873/b472ffae-a6ce-44da-a9cc-ebdea37fdc0a)
![overview](https://github.com/vercel/community/assets/174792873/9b5ddb99-75e4-45e4-954c-a9d24918ea2e)
![top paths](https://github.com/vercel/community/assets/174792873/568a42d4-6e0f-4c20-af17-e073075de6e4)

### Steps to Reproduce

N/A

**Comment by pawlean** [comment]
Hi @addda3786!

We are in the process of deploying an update to automatically compress serverless function responses, aiming to reduce FOT. This update will be fully implemented for Pro users today. However, we still need to enhance the usage page with more detailed information, such as displaying the sources of FOT usage. 

I'll let you know once this has been updated!

**Reply by addda3786** [reply]
Hi @pawlean 

Thx of the reply.
If this update is deployed today, wouldn't the FOT usage in this billing period be included as well?

[a comment by an automated account is not shown]
